flowback/ledger/models.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+from django.utils import timezone
+
+from flowback.user.models import User
+
+class Account(models.Model):
+    account_number = models.CharField(max_length=20)
+    account_name = models.CharField(max_length=100)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
refer to flowback/common/models.py, use the BaseModel to add common field created_at and updated_at